packages/components/src/Header.tsx

```diff
@@ -20,104 +21,141 @@ export const Header = ({
     setAlertMessage: () => void
     title: string
 }) => {
+    const [isButtonHovered, setIsButtonHovered] = useState(false)
+    const titleArray = title.split(/(?<=\/)/)
+
+    const UserButton = () => {
+        if (!accountId) {
+            return (
+                <Button onClick={onSignIn} variant="primary">
+                    Connect NEAR
+                </Button>
+            )
+        }
+
+        if (isButtonHovered) {
+            return (
+                <Button
+                    onClick={onSignOut}
+                    onMouseLeave={() => setIsButtonHovered(false)}
+                    variant="outline"
+                >
+                    Disconnect
+                </Button>
+            )
+        }
+
+        return (
+            <Button
+                onMouseEnter={() => setIsButtonHovered(true)}
+                variant="outline"
+            >
+                {accountId}
+            </Button>
+        )
+    }
     return (
-        <div>
-            <header
+        <Box
+            sx={{
+                width: '100%',
+                bg: 'bg',
+                borderBottom: accountId ? 0 : 1,
+            }}
+        >
+            <Container
+                as="header"
+                variant="wide"
+                pt={[2, 60]}
+                pb={[0, 36]}
+                px={0}
                 sx={{
-                    marginBottom: `1.45rem`,
-                    margin: `0 auto`,
-                    maxWidth: 960,
-                    padding: `1rem 2rem 0 2rem`,
+                    display: 'flex',
+                    flexDirection: ['column', 'row'],
+                    alignItems: ['left', 'center'],
+                    flexWrap: 'wrap',
                 }}
             >
-                {alertMessage && (
-                    <Alert>
-                        <Text>{alertMessage.substring(0, 80)} ...</Text>
-                        <Close
-                            ml="auto"
-                            mr={-2}
-                            sx={{ height: '2rem', width: '4rem' }}
-                            onClick={() => setAlertMessage('')}
-                        />
-                    </Alert>
-                )}
-                <div
+                <Box
```

Comment:
<img width="422" alt="Screenshot 2021-11-16 at 12 18 04" src="https://user-images.githubusercontent.com/5102099/141976253-8ff04bab-8493-4738-ba18-d41570296020.png">

Missing the plus button here, plus missing some space between account id and title.